apps/docs-v2/components/products/ProductServices.tsx

```diff
@@ -0,0 +1,141 @@
+import {Box, Button, Grid, Row, Typography} from '@comfortdelgro/react-compass'
+import {useMemo} from 'react'
+import {
+  AdminPortalLogo,
+  CarbyPortalLogo,
+  CMSLogo,
+  ComfortConnectLogo,
+  FMSLogo,
+  FNBLogo,
+  TEMLogo,
+  ZigStoreLogo,
+} from './ProductLogos'
+
+const ProductService = () => {
+  const lorem = 'Lorem ipsum dola sign lewded saedh kdkroe krtyer'
+
+  const products = useMemo<ProductCard[]>(() => {
```

Comment:
No `useMemo` needed and let's move it outside the component

Reply:
will do